2022q4/ci.adoc

```diff
@@ -0,0 +1,53 @@
+=== Continuous Integration
+
+Links: +
+link:https://ci.FreeBSD.org[FreeBSD Jenkins Instance] URL: link:https://ci.FreeBSD.org[https://ci.FreeBSD.org] +
+link:https://artifact.ci.FreeBSD.org[FreeBSD CI artifact archive] URL: link:https://artifact.ci.FreeBSD.org[https://artifact.ci.FreeBSD.org] +
+link:https://wiki.FreeBSD.org/Jenkins[FreeBSD Jenkins wiki] URL: link:https://wiki.FreeBSD.org/Jenkins[https://wiki.FreeBSD.org/Jenkins] +
+link:https://wiki.FreeBSD.org/HostedCI[Hosted CI wiki] URL: link:https://wiki.FreeBSD.org/HostedCI[https://wiki.FreeBSD.org/HostedCI] +
+link:https://wiki.FreeBSD.org/3rdPartySoftwareCI[3rd Party Software CI] URL: link:https://wiki.FreeBSD.org/3rdPartySoftwareCI[https://wiki.FreeBSD.org/3rdPartySoftwareCI] +
+link:https://preview.tinyurl.com/y9maauwg[Tickets related to freebsd-testing@] URL: link:https://preview.tinyurl.com/y9maauwg[https://preview.tinyurl.com/y9maauwg] +
+link:https://github.com/freebsd/freebsd-ci[FreeBSD CI Repository] URL: link:https://github.com/freebsd/freebsd-ci[https://github.com/freebsd/freebsd-ci] +
+link:https://lists.FreeBSD.org/subscription/dev-ci[dev-ci Mailing List] URL: link:https://lists.FreeBSD.org/subscription/dev-ci[https://lists.FreeBSD.org/subscription/dev-ci]
+
+Contact: Jenkins Admin <jenkins-admin@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-testing[freebsd-testing Mailing List] +
+Contact: IRC #freebsd-ci channel on EFNet
+
+The FreeBSD CI team maintains the continuous integration system of the FreeBSD project.
+The CI system checks the committed changes can be successfully built, then performs various tests and analysis over the newly built results.
+The artifacts from those builds are archived in the link:https://artifact.ci.FreeBSD.org[artifact server] for further testing and debugging needs.
+The CI team members examine the failing builds and unstable tests and work with the experts in that area to fix the code or adjust test infrastructure.
+
+During the fourth quarter of 2022, we continued working with the contributors and developers in the project to fulfill their testing needs and also keep collaborating with external projects and companies to improve their products and FreeBSD.
+
+Important completed tasks:
+
+* link:https://ci.FreeBSD.org/job/FreeBSD-main-amd64-gcc9_build/[FreeBSD-main-amd64-gcc9_build] starts sending failing reports to the committers made the commits may be related.
```

Comment:
Suggested rewrite:

* link:https://ci.FreeBSD.org/job/FreeBSD-main-amd64-gcc9_build/[FreeBSD-main-amd64-gcc9_build] now sends failing reports to the committers whose commits may be related.

Reply:
Sounds good, thanks. Please modify it directly if you're going to merge it before I back to fix this.